docs/modules/converter/pages/syntax/title.adoc

```diff
@@ -1,16 +1,49 @@
-= Title Slide Customization
-:navtitle: Title Slide
+= Title Slide
 
-The title slide is customized via AsciiDoc attributes.
-These are the global variable assigned at the top of a document under the lead
-title that look like this: `:name: value`.
+The title slide is where customization for the generated presentation can be configured via AsciiDoc xref:asciidoc:attributes:document-attributes.adoc[document attributes].
+These are the document-scope variables assigned at the top of a document, as part of the document header as xref:asciidoc:attributes:attribute-entries.adoc[attribute entries].
 
-This converter supports changing the color, image, video, iframe and
-transitions of the title slide.
+[source,asciidoc]
+----
+= Title Slide
+:revealjs_theme: sky
+:source-highlighter: highlight.js
 
-Read link:{url-revealjs-doc}#slide-backgrounds[the relevant reveal.js documentation] to understand what attributes need to be set.
-Keep in mind that for title slides you must replace `data-` with `title-slide-`.
+== Slide One
 
-See link:{url-project-examples}/title-slide-image.adoc[title-slide-image.adoc].
+* Foo
+* Bar
+* World
+----
 
-The title slide is also added a `title` CSS class to help with template customization.
+In AsciiDoc terms, the first line is the document title and the variable definitions following it are part of the xref:asciidoc:document:header.adoc[document header].
+
+Note that starting your document with a section title means AsciiDoc interprets your Document Header to be empty.
+Any variables set will not be picked up by the converter and will not influence the resulting presentation.
+The next example shows this mistake:
+
+[source,asciidoc]
+----
+== This is not a document title
+// These settings are not interpreted as Document Attributes and will not be accessible to
+// the asciidoctor-revealjs converter.
+:revealjs_theme: sky
+:source-highlighter: highlight.js
+----
+
+NOTE: Even if you start a document with a level-1 section, you can still have a document header.
+The variables then need to be defined _above_ the section header. Another way to think of this is
+having a document header without a document title.
+
+== Customizing The Title Slide
```

Comment:
I don't think "the" should be capitalized since it's not a word (but I might be wrong):

```suggestion
== Customizing the Title Slide
```

Reply:
Well, it's a word, but I understand what you mean. Can't come up with the English name for it, but in Dutch they're called "lidwoorden" and they're never capitalised in titles either.